src/wp-includes/rest-api/endpoints/class-wp-rest-posts-controller.php

```diff
@@ -369,6 +369,8 @@ public function get_items( $request ) {
 
 		$posts = array();
 
+		update_post_author_caches( $query_result );
```

Comment:
Why are we unconditionally preloading this? AFAICT, the posts controller doesn't fetch the user record for each post's author.

Reply:
`_embed` maybe?

Reply:
Because of setup post data
https://github.com/WordPress/wordpress-develop/blob/342a473e065e2bcc6d1f6b7340c633f34d6dcc38/src/wp-includes/rest-api/endpoints/class-wp-rest-posts-controller.php#L1703
https://github.com/WordPress/wordpress-develop/blob/342a473e065e2bcc6d1f6b7340c633f34d6dcc38/src/wp-includes/class-wp-query.php#L4429
https://github.com/WordPress/wordpress-develop/blob/342a473e065e2bcc6d1f6b7340c633f34d6dcc38/src/wp-includes/class-wp-query.php#L4489